### PR TITLE
Added CSS to Dark Mode to Breadcrumbs

### PR DESCRIPTION
**Pull request by bvi1994**
PLEASE SQUASH

Added CSS to Breadcrumbs so dark mode can be enabled.

### CODE REVIEW

File: src/less/breadcrumbs/ds4/breadcrumbs.less [hunk 1]
```diff
@@ -7,4 +7,11 @@
 @breadcrumbs-item-focus-hover-color: @color-core-blue;
 @breadcrumbs-item-current-color: @color-core-gray-dim;
 
+// CSS Custom Properties for DS6 only
```
Comment (ianmcburnie) [inline src/less/breadcrumbs/ds4/breadcrumbs.less:10]: You can remove the properties from this DS4 file. We are doing custom properties and dark mode for DS6 only.

Comment (ianmcburnie) [reply src/less/breadcrumbs/ds4/breadcrumbs.less:10]: Actually, I see they are missing from the ds6 file, so just move these over to ds6.

Comment (bvi1994) [reply src/less/breadcrumbs/ds4/breadcrumbs.less:10]: Doh! I put it in the wrong place! Will put it in the right place.

Comment (bvi1994) [reply src/less/breadcrumbs/ds4/breadcrumbs.less]: And fixed!

File: src/less/breadcrumbs/ds6/breadcrumbs.less [hunk 2]
```diff
@@ -7,4 +7,14 @@
 @breadcrumbs-item-focus-hover-color: @color-link-default;
 @breadcrumbs-item-current-color: @color-grey5;
 
+// CSS Custom Properties for DS6 only
+
+@media (prefers-color-scheme: dark) {
+    .switch {
+        --breadcrumbs-item-color: @color-text-default-dark-mode;
+        // --breadcrumbs-item-current-color: TBD for dark mode
+        --breadcrumbs-item-focus-hover-color: @color-link-default-dark-node;
```
Comment (ianmcburnie) [inline src/less/breadcrumbs/ds6/breadcrumbs.less:16]: Typo: `dark-node` :-)

File: src/less/mixins/utility/utility-mixins.less [hunk 3]
```diff
@@ -22,3 +22,8 @@
     text-overflow: ellipsis;
     white-space: nowrap;
 }
+
+.customProperty(@property, @propertyValue){
```
Comment (ianmcburnie) [inline src/less/mixins/utility/utility-mixins.less:26]: Nice mixin. Do you want to try using it in this PR?

File: src/less/breadcrumbs/ds6/breadcrumbs.less (hunk 2 above)
Comment (bvi1994) [reply src/less/breadcrumbs/ds6/breadcrumbs.less:16]: ~~You did not see that typo~~

Comment (ianmcburnie) [reply src/less/breadcrumbs/ds6/breadcrumbs.less:16]: I did not see that typo

movealng

File: src/less/mixins/utility/utility-mixins.less (hunk 3 above)
Comment (bvi1994) [reply src/less/mixins/utility/utility-mixins.less]: Yep, already applied it to Breadcrumb

File: dist/breadcrumbs/ds4/breadcrumbs.css [hunk 4]
```diff
@@ -1,6 +1,5 @@
 nav.breadcrumb,
 nav.breadcrumbs {
-  color: #333;
```
Comment (ianmcburnie) [inline dist/breadcrumbs/ds4/breadcrumbs.css:3]: Hmm, something's not right, we lost the color styles in the css.

Comment (bvi1994) [reply dist/breadcrumbs/ds4/breadcrumbs.css:3]: I don't remember modifying the file at all. Do you think I should just manually add back the colors in dist?

Comment (bvi1994) [reply dist/breadcrumbs/ds4/breadcrumbs.css:3]: In fact, I never touched anything in dist up until then. It seems that modifying the file manually does the trick but I'm seeing if there's a better way.

Comment (bvi1994) [reply dist/breadcrumbs/ds4/breadcrumbs.css:3]: Upon further investigation, running `yarn build` removes the colors in `dist`, even when I manually add them.

**Comment by bvi1994** [comment]
I'm gonna work on the customProperties mixin at a later time. I did include in the utilities files commented out so I can easily work on it at a later time. For now, the values for breadcrumb is hard coded.

**Comment by ianmcburnie** [comment]
I'm going make a couple of quick changes directly to your branch, then will merge.

**Comment by ianmcburnie** [comment]
I fixed the mixin. @agliga this was the mixin suggested by you initially. Please take a quick look.

**Comment by ianmcburnie** [comment]
Also just noticed that we are missing the default, light-mode variables in the ds6 file. I will add those now.

**Comment by ianmcburnie** [comment]
@agliga Need this and wizard stepper for my demo tomorrow, so I'm going to merge now and let me know if you spot anything later.

**Comment by bvi1994** [comment]
Woo hoo! My first contribution for eBay is now official! :)

**Comment by ianmcburnie** [comment]
Level up!

**Pull request merged**